modelgym/guru.py

```diff
@@ -155,6 +156,18 @@ def check_class_disbalance(self, y):
 
         return candidates
 
+    def draw_correlation_heatmap(self, X, feature_indexes=None):
+        if feature_indexes is None:
```

Comment:
Why feature index instead of feature column(s)?

Reply:
To make subscriptions on graphs and to release a user from writing _get_feature by himself. I think, it's useful in case of list of list.

Reply:
keeping track of column indexes is difficult since those can change during array transformation.. let's treat 'feature_indexes' either as list of integers or list of strings in which case you should resolve it to index inside the method.